### PR TITLE
full responsive media page with no bootstrap

### PR DESCRIPTION
**Pull request by itsnamangoyal**
> media-responsive branch
>Screenshot from 2020-03-16 05-24-22

> develop branch
>Screenshot from 2020-03-16 05-25-57

### CODE REVIEW

**Comment by avantikabharti** [comment]
image
I see hindi text is overflowing out of the container in mobile view. Please make the required changes

File: frontend/src/components/Pages/Media/Media.css [hunk 1]
```diff
@@ -20,45 +24,72 @@ a.nostyle:link {
...
+.newsBoxTopContent{
+  width: 75%;
+  display: flex;
+  vertical-align: middle;
+  justify-content: center;
+  flex-flow: column nowrap;
+  padding: 4%;
 }
 
-@media only screen and (min-device-width : 768px) and (max-device-width : 1024px) {
```
Comment (avantikabharti) [inline frontend/src/components/Pages/Media/Media.css:53]: A basic responsive web app guideline, media queries should use in order to make app responsive not the Percentage.

Comment (itsnamangoyal) [reply frontend/src/components/Pages/Media/Media.css:53]: @avantikabharti  added media queries to for the news box class. but still using fluid for everything else.

Comment (itsnamangoyal) [reply frontend/src/components/Pages/Media/Media.css:53]: also added extra margin to avoid overflow of hindi text.

**Comment by itsnamangoyal** [comment]
@shankarsingh482

**Pull request merged**